encoding/rs/encode.go

```diff
@@ -77,29 +78,45 @@ func (g *Encoder) MakeFrames(
 	indices := make([]uint32, 0)
 	frames := make([]Frame, g.NumChunks)
 
-	for i := uint64(0); i < uint64(g.NumChunks); i++ {
+	numWorker := uint64(g.NumRSWorker)
```

Comment:
No need to convert to uint64, since L83 makes the conversion already.
Also it probably has no need to use that many workers.

Reply:
it is assigned to number chunks at L84